src/odemis/acq/stream/_helper.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, name, detector, dataflow, emitter, scanner=None, sstage=None,
         # We ensure in the setters that all the data is always consistent:
         # roi set: roi + pxs → repetition + roi + pxs
         # pxs set: roi + pxs → repetition + roi (small changes)
-        # repetition set: repetition + roi + pxs → repetition + pxs + roi (small changes)
+        # repetition set: repetition + roi → repetition (adjusted) + pxs + roi (small changes)
 
```

Comment:
I think this is very hard to understand. What does it mean "We ensure in the setters that all the data is always consistent:" and what is the rep set?